Documentation/Specification/commands.tex

```diff
@@ -743,13 +758,17 @@
 the command menu item associated with the gesture and the command table in which
 it was found.  Note that \arg{gesture} may be either a keyboard gesture name of
 a gesture object, and is handled in the same way as in \cl{find-keystroke-item}.
+
 \ReadOnly
 
 Unlike \cl{find-keystroke-item}, this follows the sub-menu chains that can be
-created with \cl{add-menu-item-to-command-table}.  If the keystroke accelerator
-cannot be found in the command table or any of the command tables from which it
-inherits, \cl{lookup-keystroke-item} will return \cl{nil}.  \arg{command-table}
-is a \term{command table designator}.
+created with \cl{add-menu-item-to-command-table}. If the sub-menu has an
```

Comment:
Maybe "If a given sub-menu" instead of "If the sub-menu"?

Reply:
fixed